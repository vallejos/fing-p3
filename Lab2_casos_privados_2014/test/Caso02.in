7
3 -1
0 -1
0 1 -1
2 4 -1 
5 6 -1
4 6 -1 
4 5 -1
Antropologia 10 Flora 5 Musica 9 -1 -1
Antropologia 11 Flora 3 Musica 10 -1 -1
Antropologia 7 Flora 4 Musica 6 -1 -1
Antropologia 4 Flora 2 Musica 3 -1 -1
Antropologia 7 -1 -1
Antropologia 5 -1 -1
Antropologia 3 Filosofia 20 -1 -1
identificarColecciones 4 Antropologia Flora Musica Filosofia
identificarColecciones 4 Musica Antropologia Flora Filosofia
identificarColecciones 4 Flora Antropologia Filosofia Musica
salir
