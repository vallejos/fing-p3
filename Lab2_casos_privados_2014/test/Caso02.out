Tarea 2 - 2014 - Programacion 3.

>0 -> Musica
1 -> Antropologia
>0 -> Musica
1 -> Antropologia
>0 -> Musica
1 -> Antropologia
>FIN.
